cheatsheets/Abuse_Case_Cheat_Sheet.md

```diff
@@ -113,15 +105,15 @@ It is important to take into account **Technical** and **Business** kind of abus
 *Example:*
 
 - Technical flagged abuse case: Add Cross Site Scripting injection into a comment input field.
-- Business flagged abuse case: Ability to modify  the price of an article in an online causing the user to pay a lower amount for the wanted article.
+- Business flagged abuse case: Ability to modify the price of an article in an online causing the user to pay a lower amount for the wanted article.
 
 ## When to define the list of Abuse Cases?
 
-On an agile project, the definition workshop must be made after the meeting in which User Stories are includen in a Sprint.
+On an Agile project, the definition workshop must be made after the meeting in which User Stories are included on a Sprint.
```

Comment:
- `In Agile projects`. To follow how Waterfall is written, or the opposite, if that looks better to you.
- included **in** a sprint, and not **on**